dashboard/test/models/concerns/curriculum/course_types_test.rb

```diff
@@ -48,15 +48,13 @@ class CourseTypesTests < ActiveSupport::TestCase
     assert @unit_facilitator_to_teacher.pl_course?
     assert @unit_universal_instructor_to_teacher.pl_course?
     assert @unit_plc_reviewer_to_facilitator.pl_course?
-    assert @unit_universal_instructor_to_teacher.pl_course?
```

Comment:
Looks like I was checking the same things twice in a bunch of places in this file